Little edit to SwiftUI: Toggle (#1507)
Codecademy/docs: content/swiftui/concepts/views/terms/toggle/toggle.md

The following parameters can be applied to a `Toggle` view:

> - The `"Title String"` parameter is used instead of a [closure](https://www.codecademy.com/resources/docs/swift/closures).
> - `"Title String"` is an optional paramenter; Based on your usecase you can decide to either enter it when calling `Toggle` or inside a [closure](https://www.codecademy.com/resources/docs/swift/closures), as shown in the example below.

Reviewer: ```suggestion
- `"Title String"` is an optional parameter. Based on the use-case, this can be applied when calling `Toggle` or used inside a [closure](https://www.codecademy.com/resources/docs/swift/closures), as shown in the example below.
```
